src/Hl7.Fhir.Base/Specification/Source/ResolverException.cs

```diff
@@ -0,0 +1,55 @@
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Hl7.Fhir.Specification.Source;
+
+public class ResolverException : CodedException
```

Comment:
Make sure you document public classes and members - at least the new ones in this PR that you have introduced!